recipes/hypre/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hypre" %}
+{% set version = "2.11.2" %}
+{% set sha256 = "25b6c1226411593f71bb5cf3891431afaa8c3fd487bdfe4faeeb55c6fdfb269e" %}
+{% set mpi = 'mpich' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/LLNL/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ mpi }}
+    - openblas 0.2.19
+    - cmake
+    - toolchain
```

Comment:
You could drop the explicit specification of `toolchain`. It's getting pulled as a build time dependency of `openblas`. It's not actually hurting anything, just unnecessarily specifying dependencies.